### PR TITLE
Writing tutorials: gamma calculation

### PR DESCRIPTION
**Pull request by nlambriICH**
Opening this pull request as per issue #1634.

I need a general feedback about the structure of the tutorial, and what I should change/improve.

I also need some clarification of what are the _Current distance_ and _RAM split count_ which are shown in the log of the PyMedPhys gamma. I think I got the first, but I cannot figure out what the latter means.

Thanks in advance for your time!

### CODE REVIEW

**Comment by Matthew-Jennings** [comment]
Hi @nlambriICH,

Looks like a perfectly reasonable start, thank you! I'll have a more in-depth look soon. As for structure... well, at the moment I would probably say you have the freedom to structure in the way that you think is best. To be honest, the biggest gain probably comes from just have _something_ with sufficiently deep explanation to which users can refer. It need not be perfect in its first iteration, and I suspect it's premature to set a rigid template until we have a concerted effort to provide tutes.

@SimonBiggs, upon reviewing this, I can't help but be reminded of the whole coords/axes discussion and an old plan to adjust things. :D Not just the old coord system point, but also the point of using explicit "axes" variables rather than coord meshgrids (that you can just slice if you want the axes). Probably 1small points that don't matter too much - especially with a tute!

Also, now that I have a working home computer and thoroughly miss developing stuff, maybe I can dedicate some time to speeding up gamma.

**Comment by Matthew-Jennings** [comment]
@SimonBiggs, does ReviewNB allow for running/seeing cell outputs? I can't work out how...

**Comment by SimonBiggs** [comment]
> @SimonBiggs, does ReviewNB allow for running/seeing cell outputs? I can't work out how...

No it doesn't, but google colab does:

https://colab.research.google.com/github/pymedphys/pymedphys/blob/39739c8dbc7f9f4e9488a34c00c7251a549385d6/lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

A given notebook for a given git commit can be turned into a colab address by swapping "https://github.com/" for "https://colab.research.google.com/github/". Eg:

https://github.com/pymedphys/pymedphys/blob/39739c8dbc7f9f4e9488a34c00c7251a549385d6/lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

turns into:

https://colab.research.google.com/github/pymedphys/pymedphys/blob/39739c8dbc7f9f4e9488a34c00c7251a549385d6/lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

If you were interested in creating something to help ease contributions and review I could see some significant benefit in making a GitHub Action bot that detects whether or not an ipynb has changed and then commenting in the PR with a commit hash based colab link...

**Comment by SimonBiggs** [comment]
> @SimonBiggs, upon reviewing this, I can't help but be reminded of the whole coords/axes discussion and an old plan to adjust things. :D Not just the old coord system point, but also the point of using explicit "axes" variables rather than coord meshgrids (that you can just slice if you want the axes). Probably 1small points that don't matter too much - especially with a tute!

Yeah, probably best to spin that off into a separate issue. And also likely something I don't really have the bandwidth for. But, more than happy for you to give it a crack. This comes with the caveat that pymedphys is stabilising and moving forward we need to be prioritising backwards compatibility guarantees.

**Comment by SimonBiggs** [comment]
@nlambriICH this is amazing :slightly_smiling_face: :tada: :balloon:

**Comment by SimonBiggs** [comment]
@nlambriICH, something I really like with this tutorial is that you have flagged the investigatory approach of having a look at the dose difference. And flagging that a significant dose difference, with higher gamma values, would result in it taking longer.

To answer your questions:

> I also need some clarification of what are the _Current distance_ and _RAM split count_ which are shown in the log of the PyMedPhys gamma. I think I got the first, but I cannot figure out what the latter means.

> Current distance: 0.50 mm | Number of reference points remaining: 128 | Points tested per reference point: 348 | RAM split count: 111:
> Current distance (?): the current distance between reference grid point and evaluated grid points
Number of reference points remaining: the number of points left to be evaluated in the reference dose grid
Points tested per reference point: the number of points considered in the evaluation dose grid for the current reference point
RAM split count: (?)

Yup, certainly. Yes, you have the current distance correct. Essentially the calculation iteratively steps further and further away. At each iteration points are dropped out of the calculation. This "dropping" is achieved because for a certain gamma value there exists a distance separation where even perfect dosimetric agreement would result in a higher gamma. Therefore, the calculation for that point can stop. For 3D gamma the distance can be thought of the radius of the shell of points that is currently being interpolated.

The RAM split count is supposed to say "1" there. Unfortunately the way it is printing there the line is not clearing when it logs the output. I think this would likely be better not clearing the output at all, and instead allowing people to scan back through the history of the calculation. @Matthew-Jennings, is this a fix you might be interested in implementing?

Essentially, if there are a lot of points being calculated and they are not fitting into the designated RAM assigned to the calculation then it has to do the calculation in stages. A number greater than 1 here can tell you that it it is taking that times as much slower, potentially unecessarily.

**Comment by nlambriICH** [comment]
@SimonBiggs thanks for the details, I'll add them to the tute!

I see that @Matthew-Jennings has started working also on the loggin of the gamma, so I think I'll commit the next iteration of this tutorial when the changes are complete (as I think it will be done soon, correct me if not).

**Comment by SimonBiggs** [comment]
> so I think I'll commit the next iteration of this tutorial when the changes are complete (as I think it will be done soon, correct me if not).

Sounds like a good approach to me 🙂

**Comment by nlambriICH** [comment]
I have a few points to clarify:
@Matthew-Jennings I see you reduced the decimal places of "Global dose threshold", but maybe it's better to keep one more decimal (so up to three) so that the value is more clearly linked to the global normalisation. What do you think? 

@SimonBiggs do we need to keep the pip install at the beginning of the notebook? At the moment I've left it commented out

Also, what do you think of the title of tutorial? Should it be more specific?

**Comment by Matthew-Jennings** [comment]
> I have a few points to clarify: @Matthew-Jennings I see you reduced the decimal places of "Global dose threshold", but maybe it's better to keep one more decimal (so up to three) so that the value is more clearly linked to the global normalisation. What do you think?

Yep, more than happy to have three. Feel free to change to three and include in this PR :) Or I can do it if you prefer.

**Comment by nlambriICH** [comment]
@Matthew-Jennings Perfect, I'll push the changes once I have also the feedback of Simon

**Comment by SimonBiggs** [comment]
> Perfect, I'll push the changes once I have also the feedback of Simon

Happy for you both to run ahead without me :) keen to structure things so that eventually no one person is a point of failure (keyman risk / bus factor) for the success of the library.

Regarding the pip install, I don't think it'd hurt to have a discussion that details the use `!pip install pymedphys` and then has it commented out and explains that it can be uncommented to install (might have to also include pydicom if going that route). Also don't mind it not being there at all. I see pros and cons of both and happy for you to make the judgement call.

File: .github/workflows/library.yml
Comment (SimonBiggs) [inline .github/workflows/library.yml:362]: Oh, that's cool :slightly_smiling_face:. Thanks for fixing that :slightly_smiling_face:

Comment (nlambriICH) [reply .github/workflows/library.yml:362]: I was also trying to fix the failing tests.

The problem of _Library / Tests (windows, docs) (pull_request)_ seems related to poetry with python 3.9, and should be fixed by [these changes](https://github.com/python-poetry/poetry/pull/5645) when they will be available.

**Comment by Matthew-Jennings** [comment]
Sorry for the delay in getting to this, @nlambriICH. I should be able to look into it carefully today. Thank you for your efforts and patience!

File: lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb
Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <blockquote>The PyMedPhys&nbsp;<a href="https://github.com/pymedphys/pymedphys/raw/c029675cd7d66b663c9ab1971b55fa38be5c38cc/lib/pymedphys/docs/lib/ref/gamma.rst" target="_blank">gamma</a>&nbsp;<strong>tool </strong>allows <strong>the user</strong> to compute <s>the </s>gamma (γ) index <strong>distributions </strong>between a reference and an evaluation dose distribution in 1D, 2D and 3D <strong>and obtain gamma pass rates for these distributions</strong>. <strong>The tool features several options for optimising the computation for the user's application.</strong></blockquote><p> </p><blockquote>In this tutorial, you will compute the <strong>gamma </strong>pass rate using the PyMedPhys <a href="https://github.com/pymedphys/pymedphys/raw/c029675cd7d66b663c9ab1971b55fa38be5c38cc/lib/pymedphys/docs/lib/ref/gamma.rst" target="_blank">gamma</a> <strong>tool</strong>. You will tweak its parameters to speed up the computation of the gamma index <strong>distribution</strong>.</blockquote><p> </p><p>Also, your links to "PyMedPhys gamma" throughout the tutorial probably need to be directed to the published <a href="https://docs.pymedphys.com/lib/ref/gamma.html" target="_blank">docs</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <blockquote>Both the reference and evaluation <s>images </s><strong>datasets </strong>are 3D dose distributions.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <blockquote><s>The positional arguments of the PyMedPhys </s><a href="https://github.com/pymedphys/pymedphys/raw/c029675cd7d66b663c9ab1971b55fa38be5c38cc/lib/pymedphys/docs/lib/ref/gamma.rst" target="_blank"><s>gamma</s></a><s> specify the axes and dose of the reference and evaluation images, as well as the dose difference and distance to agreement to perform the computation. </s></blockquote><p> </p><p>This could benefit from a little more fleshing out, I think. Maybe not necessary for a first pass. Maybe for now this cell is better off saying something like:</p><p> </p><blockquote>Now, we need to extract the dosegrids as well as the x, y, and z axes from the reference and evaluation RTDOSE datasets in a form readable by the PyMedPhys gamma function: You can use <code>zyx_and_dose_from_dataset()</code> for this like so: </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <p>Wording here is a little tricky, and speaks to a limitation of the current API --&gt; we're not particularly careful about coordinate systems here. Not sure if this is true for a decubitis orientation, for example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    dose_difference = dose_evaluation - dose_reference
</pre><p>There might be some clever optimisations here, but probably unneeded.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <p>Maybe a good point to flesh out why this is the case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <p>A TODO: for another time maybe.</p><p> </p><p>We could consider a note here commenting on the shape of the distribution and how it might differ from a commercial (fast) gamma calculation tool, where certain optimisations are made to obtain an accurate pass rate quickly at the expense of computing erroneously large gamma index values, which is evident in the shape of the gamma histogram.</p><p> </p><p>Probably something more for Simon or I later. We could note it in a Github issue maybe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <blockquote><s>Once the gamma has been computed, one usually considers the passing rate to have a final evaluation of the agreement between the reference and evaluated dose distributions. </s></blockquote><p> </p><p>I'm not quite sure what you're trying to say here, but my feeling is that it is best to remove this line. For common medical physics applications, the gamma index is well known to be a useful but still <em><u>insufficient </u></em>tool for comparison between dose distributions. </p><p> </p><blockquote>The passing rate is defined as the fraction of points having&nbsp;γ ≤ 1. <s>A threshold is chosen (e.g., 95%) and the test is passed if the passing rate exceeds such threshold.</s></blockquote><p> </p><p>Not sure there's a need to comment on a threshold/tolerance pass rate. It's not part of the API and is application-specific.</p><p> </p><blockquote><s>We check if in this case the test is passed (probably you already know the answer looking at the histogram above)</s></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <blockquote>In practice, <strong>calculations of 3D gamma index distributions can be very time consuming; a problem that is further exacerbating in applications where multiple comparisons are required. </strong><s>one might need to perform computations of the gamma for several dose comparisons, which could result in a time consuming operation. </s></blockquote><blockquote><br></blockquote><blockquote><strong>For many applications, it is sufficient to simply calculate an accurate</strong> <s>In case we need only the final </s>pass rate.</blockquote><blockquote><br></blockquote><blockquote><strong>If we are uninterested in accurate gamma index values for gamma &gt; 1, we can suppress calculations for gamma values above 1 by passing a sutiable a value (e.g. 1.1) to the </strong><code><strong>max_gamma</strong></code><strong> parameter of the gamma function. </strong></blockquote><blockquote><strong>Moreover, random sampling of a subset of the dose distributions for gamma index calculation may also produce a sufficiently accurate pass rate. This can also introduce significant decreases in calculation time. We can take compute for a random subset by providing a suitable fraction to the </strong><code><strong>random_subset</strong></code><strong> parameter of the gamma function. </strong></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    valid_gamma = gamma[~np.isnan(gamma)]
</pre><p>Maybe we should explain this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <p>I can't see the outputs here, but a few percent is not small. It would be nice to find an example where the pass rate changes by less than 1%. Maybe by using 1/10 of the points?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (Matthew-Jennings) [inline lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <p>Overall this is a very nicely put together tutorial! Great structure, great, relevant content that is to the point. Thank you! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

**Comment by nlambriICH** [comment]
Thanks @Matthew-Jennings for the helpful comments!

I didn't have access to my PC last week. I plan to push the next iteration of the tutorial by the end of this week.

File: lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb
Comment (nlambriICH) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: I agree. Also, I think it's more readable.

Comment (nlambriICH) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: I don't think I got your point... Are you referring to this sentence? 

> <code>axes_reference</code> and <code>axes_evaluation</code> are tuples of numpy arrays, whose values define the (z, y, x) grid coordinates in the DICOM frame of reference.

Comment (nlambriICH) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: I can't see the cell you refer to in this comment.

Comment (nlambriICH) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: > > Once the gamma has been computed, one usually considers the passing rate to have a final evaluation of the agreement between the reference and evaluated dose distributions.
> 
> I'm not quite sure what you're trying to say here, but my feeling is that it is best to remove this line. For common medical physics applications, the gamma index is well known to be a useful but still _insufficient_ tool for comparison between dose distributions.

Here I'm trying to move to the "topic" of gamma pass rate, as the focus of the tute should be on how to speed up the gamma pass rate computation (where there is no need for an exact gamma index distribution). Maybe I could point out that the gamma index is still an insufficient tool for comparison between dose distributions, as you've mentioned.

> Not sure there's a need to comment on a threshold/tolerance pass rate. It's not part of the API and is application-specific.

Yes, I agree it's application-specific, but I was just trying to give an example on how the pass rate can be used. I think it might be useful to someone who's not familiar with this.

Comment (nlambriICH) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: I tried with 1/10 and got differences of less than 1%

Comment (nlambriICH) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: Sure, I'll add a comment

Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: @nlambriICH if you click on the following link:

image

and then scroll down to this section of the review:

image

Hopefully that helps?

Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: (You might have to log in with your GitHub account first)

Comment (Matthew-Jennings) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <p>I understand. My first point is centred on the specific wording: "<em>considers the passing rate to have a </em><strong><em>final </em></strong><em>evaluation</em>". To me, this implies (admittedly softly) that a pass rate is a sufficient metric for all similar dose comparison tasks. I'm not suggesting that we do not cover the topic of gamma pass rate in the tutorial, I'm suggesting that we make no comment about the <em>utility </em>of the gamma pass rate at all.</p><p> </p><p>Similarly, for the second paragraph, I'm just trying to avoid any specific wording that implies that a 95% pass rate is some objective, general/well-accepted value. I'd prefer if we didn't even have a faint hint of opinion on this in PyMedPhys, and instead leave such decisions entirely to the user.</p><p> </p><p>Maybe something like the following could replace the text in the cell?</p><p> </p><blockquote>Once one has computed the gamma index distribution, one can compute a gamma pass rate: the fraction of data elements for which γ ≤ 1.</blockquote><blockquote>Suppose that, for our purposes, we consider the comparison to have "passed the test" if the gamma pass rate is above <strong>95%</strong> Let's check if the test passes:</blockquote><p> </p><p>It's a small point that I won't push. Just trying to explain the reasoning for my comments. :)</p>

Comment (Matthew-Jennings) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <p>Great!</p>

Comment (Matthew-Jennings) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <p>I'll check the code, but I'm not sure that <code>zyx_and_dose_from_dataset()</code> actually returns a tuple of axes that are always strictly in the order <code>z, y, x</code>. For example, if the scan orientation is <em>Head First Decubitis Left</em>. If it's not, then we <em>could</em> try to be super specific about the wording (i.e., that for <em>this</em> case, "<em>the dose image is composed of...</em>"). But honestly, I wouldn't worry for now.</p>

**Review by Matthew-Jennings — APPROVED** [review]
I'll approved this for merge. The outstanding comments are not critical; address at your discretion. :)

Thank you for this contribution, @nlambriICH !

**Comment by nlambriICH** [comment]
> I'll approve this for merge. The windows doctests are fixed in another PR.
> 
> The outstanding comments are not critical; address at your discretion. :)
> 
> Thank you for this contribution, @nlambriICH !
> 
> EDIT: @SimonBiggs: @nlambriICH probably won't be able to merge while the windows doctests are failing. It's greyed out for me. Same deal for #1622.

Thank you @Matthew-Jennings for the feedback!

Before the merge I'd like to address the remaining comments (by the end of this week) :)

File: lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb
Comment (nlambriICH) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: <p>I'm going to make the text more "generic" about this point then</p>

Comment (nlambriICH) [reply lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb:1]: I agree, I'll be more vague about this. Let me know if the new phrasing works better

**Comment by SimonBiggs** [comment]
> Before the merge I'd like to address the remaining comments (by the end of this week) :)

Also @nlambriICH, given @Matthew-Jennings has left his approval, you're free to merge whenever you're ready. Of course if you make any significant changes beyond what has been reviewed already it would be worth flagging it for a re-review.

**Comment by nlambriICH** [comment]
@SimonBiggs The last changes I made are minor. I'd like to merge this PR but it seems I don't have the rights
merge

**Comment by SimonBiggs** [comment]
> I'd like to merge this PR but it seems I don't have the rights

Thanks for flagging that :slightly_smiling_face:.

Are the permissions fixed now?

**Comment by nlambriICH** [comment]
Fixed, thanks! I'm going to squash and merge now 🙂

**Comment by SimonBiggs** [comment]
No need to squash. Much prefer keeping the history in tact really...

**Comment by nlambriICH** [comment]
I'm sorry, I've just merged... I thought the squash would be better as there were lots of test commits to try fix the CI...

**Comment by SimonBiggs** [comment]
All good don't stress :slightly_smiling_face:.

By the way! Congrats on the first complete PR into PyMedPhys :slightly_smiling_face: :tada: :balloon: :firecracker: :rocket:. I reckon it's overdue to have the next PR be adding yourself to the team page over at:

https://github.com/pymedphys/pymedphys#our-team

@sjswerdloff would you be in a position to help @nlambriICH if he needs any help getting that team PR + logo completed?

Logo needs to be a png file that is 200x200 px.

**Comment by sjswerdloff** [comment]
> All good don't stress :slightly_smiling_face:.
> 
> By the way! Congrats on the first complete PR into PyMedPhys :slightly_smiling_face: :tada: :balloon: :firecracker: :rocket:. I reckon it's overdue to have the next PR be adding yourself to the team page over at:
> 
> https://github.com/pymedphys/pymedphys#our-team
> 
> @sjswerdloff would you be in a position to help @nlambriICH if he needs any help getting that team PR + logo completed?
> 
> Logo needs to be a png file that is 200x200 px.

Yup, happy to help if needed

**Pull request merged**
